code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -736,7 +736,7 @@
 /datum/reagent/toxin/spewium/on_mob_life(mob/living/carbon/C)
 	.=..()
 	if(current_cycle >=11 && prob(min(50,current_cycle)))
-		C.vomit(10, prob(10), prob(50), rand(0,4), TRUE, prob(30))
+		C.vomit(10, prob(10), prob(50), rand(0,4), TRUE)
```

Comment:
This is a bit of an odd line, what was it originally intended to do? Do you know?

Reply:
That lines up with the toxic variable on the vomit proc; so I guess, 30% chance of making the vomit toxic. I guess so that it doesn't always self-purge, or something?

Reply:
The toxic param did nothing at all, so i just removed it. I think there used to be toxic and non-toxic variants of the icon, but they seem to have been removed at some point.

Reply:
Yeah I think toxic vomit was green. Since the blood/stomach split, that behaviour was probably changed. I think normal vomiting used to reduce nutrition but not remove reagents, whereas toxic vomit would remove reagents from your blood.